docs/UserGuide.md

```diff
@@ -72,12 +72,53 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Finding a person by fields: `find`
+
+Finds persons whose fields match the given keywords.
+
+Format: `find [n/KEYWORD …] [a/KEYWORD …] [e/KEYWORD …] [p/KEYWORD …] `
+
+* There should be at least one field i.e. `find` is an invalid command.
+* The search is case-insensitive. e.g`hans` will match `Hans`
```

Comment:
Some e.g. end with a period while others don't, and the e.g on line 82 is missing a space after it